### PR TITLE
chore(cli): preinstall throw on node < 10

### LINKED ISSUE #4035: Detect and enforce Node Version in `preinstall` hook so pnpm fails clearly
Discussed from https://github.com/prisma/prisma/issues/3930
https://prisma-company.slack.com/archives/CEYCG2MCN/p1602759196130900

In the CLI in the preinstall hook we could use something like 
```
const nodeVersions = process.version.split('.')
const nodeMajorVersion = parseInt(nodeVersions[0].slice(1))
const nodeMinorVersion = parseInt(nodeVersions[1])

if (nodeMajorVersion < 10) {
// exit with error
```

Yarn doesn't need this because Yarn reads the package.json and error if the Node version required doesn't match, while npm is supposed to print a warning only.

### PR DESCRIPTION
**Pull request by williamluke4**

### CODE REVIEW

**Comment by Jolg42** [comment]
Did you try to run the code with old Node version, maybe take a screenshot?

**Comment by williamluke4** [comment]
image

File: src/packages/cli/scripts/preinstall.js [hunk 1]
```diff
@@ -53,6 +53,15 @@ const b = (str) => BOLD + str + RESET
 const white = (str) => WHITE_BRIGHT + str + RESET
 
 export function main() {
+  const nodeVersions = process.version.split('.')
+  const nodeMajorVersion = parseInt(nodeVersions[0].slice(1))
+  debug(`Node Version: ${nodeMajorVersion}`)
+  if (nodeMajorVersion < 10) {
+    console.error(
+      drawBox({ str: `Prisma only supports node >= 10`, verticalPadding: 1, horizontalPadding: 3 }),
```
Comment (Jolg42) [inline src/packages/cli/scripts/preinstall.js:61]:
```suggestion
      drawBox({ str: `Prisma only supports Node.js >= 10`, verticalPadding: 1, horizontalPadding: 3 }),
```

**Review by Jolg42 — APPROVED** [review]
👍

**Pull request merged**